src/styles/custom.scss

```diff
@@ -1,14 +1,79 @@
+@import "~bootstrap/scss/bootstrap";
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,500;1,700&display=swap');
+
 $body-bg: #F8F9FA;
 $montserrat: 'Montserrat', sans-serif;
 $lora: 'Lora', serif;
 $open-sans: 'Open Sans', sans-serif;
 $main-font-color: #0C2028;
+$body-bg: #F8F9FA;
+$secondary-font-color: #2F4550;
+$italic-color: #537C8C;
 $primary: #82B29A;
 $orange: #E17B60;
 
 body {
-    font-family: $montserrat;
     color: $main-font-color;
+    font-family: $montserrat;
```

Comment:
The base body font is not Montserrat, it's Open Sans / 16px. We probably also don't need to explicitly define letter-spacing for our base body font, as long as it looks okay in practice. Also, keeping the body background color here is fine, but just make sure to alphabetize the style properties in this file.

Reply:
This is what Alison brought up in the meeting on Tuesday. She wanted to make sure the letter spacing was correct because it did not match what she wanted. I just updated it to match the design files. I removed the font-family from the body